src/components/nav/nav.tsx

```diff
@@ -0,0 +1,116 @@
+import { Component, Element, Event, EventEmitter, h, Host, Prop, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-nav",
+  styleUrl: "nav.scss",
+  shadow: true
+})
+
+/**
+ * @slot logo - A slot for adding a `calcite-logo` component to the primary nav level.
+ * @slot user - A slot for adding a `calcite-user` component to the primary nav level.
+ * @slot progress - A slot for adding a `calcite-progress` component to the primary nav level.
+ * @slot primary-content-start - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the start position of the primary nav level.
+ * @slot primary-content-center - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the center position of the primary nav level.
```

Comment:
Do we need to say `primary-content-center` or could we just do `primary-content`?

Reply:
since we have both `primary-content-start` & `primary-content-end` , i think its better to be descriptive by adding `content` at the end.

Reply:
It's inconsistent with list-item though.

https://github.com/Esri/calcite-components/blob/3b2f4a0377ff63d67f42b43e28b1bbe8b2b7d251/src/components/list-item/list-item.tsx#L43-L47

Reply:
Because `content-center` isn't the most common slotting location, having the extra `-center` makes sense in this case, IMO. In List, (Modal, etc.), the `content` slot is "the main area".

Reply:
@jcfranco what do you think?

Reply:
After some back and forth, we'll be moving forward with `x-content-center`. Reasoning:

* unlike list-item, any slot can be used as the main content slot
* center slot is purely placing and not replacing content (compared to list-item)
* `content` appears to be the most common/appropriate word for that area in navigation bars
* `center` matches CSS terminology (along with `start`/`end`)